gnmvidispine/vs_item.py

```diff
@@ -798,14 +798,15 @@ def placeholder_adopt(self, file_ref, shape_tag='original', priority='MEDIUM'):
         else:
             logging.info("placeholder adopt returned {rtn}".format(rtn=rtn))
 
-    def import_base(self,shape_tag='original', priority='MEDIUM', essence=False, thumbnails=True, jobMetadata=None):
+    def import_base(self,shape_tag='original', priority='MEDIUM', essence=False, thumbnails=True, jobMetadata=None, noTranscode=False):
         """
         prepares arguments for an import call. This is an internal method, called by import_to_shape and streaming_import_to_shape
         :param shape_tag: shape tag to import to. Defaults to 'original'
         :param priority: priority for import job. Defaults to 'Medium'
         :param essence: is this an essence update or not
         :param thumbnails: should Vidispine re-create thumbnails or not
         :param jobMetadata: Dictionary of key/values for the job metadata - see Vidispine import documentation for details
+        :param noTranscode: Should Vidispine not attempt to transcode the data. Defaults to 'False'.
```

Comment:
I'd word this something like "If True, prevents Vidispine from attempting to transcode the imported item"